server/channels/store/sqlstore/channel_store.go

```diff
@@ -3113,6 +3113,20 @@ func (s SqlChannelStore) GetMembersForUserWithPagination(userId string, page, pe
 	return dbMembers.ToModel(), nil
 }
 
+func (s SqlChannelStore) GetMembersForUserWithCursorPagination(userId string, perPage int, fromChannelID string) (model.ChannelMembersWithTeamData, error) {
+	dbMembers := channelMemberWithTeamWithSchemeRolesList{}
+	err := s.GetReplica().Select(&dbMembers, channelMembersWithSchemeSelectQuery+"WHERE ChannelMembers.UserId = ? AND ChannelId > ? ORDER BY ChannelId ASC Limit ?", userId, fromChannelID, perPage)
```

Comment:
Should this not use the query builder?

Reply:
For simple cases like this, it's not necessary. A query builder should be used when you have multiple if conditions that can change different parts of the SQL Query. In this case, it's a static query with parameters. So no need for that.

Reply:
Seeing a `+` in a SQL query did scare me a bit even though the arguments are added in a safer manner, but I'll trust your judgment here